linux/fedora-rpm/audacity.cfg

```diff
@@ -0,0 +1,6 @@
+config_opts['releasever'] = '34'
+config_opts['target_arch'] = 'x86_64'
+config_opts['legal_host_arches'] = ('x86_64',)
+config_opts['use_host_resolv'] = False
+
+include('templates/fedora-branched.tpl')
```

Comment:
I hope that this config disables network inside the chroot. It is very useful to test builds without access to network.

Reply:
The docs say it is and I observed, that it does